pynamodb/connection/base.py

```diff
@@ -178,7 +179,8 @@ class Connection(object):
     A higher level abstraction over botocore
     """
 
-    def __init__(self, region=None, host=None, session_cls=None):
+    def __init__(self, region=None, host=None, session_cls=None,
+                 request_timeout_seconds=None, max_retry_attempts=None, base_backoff_ms=None):
```

Comment:
Actually I have implemented it as max_retry_attempts.

Let's say you pass 
max_retry_attempts = 0. There will be no retries.
max_retry_attempts = 1. There will be one retry. So totally two calls.